google-cloud-dns/google-cloud-dns.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.0.0"
 
   gem.add_dependency "google-cloud-core", "~> 1.2"
-  gem.add_dependency "google-api-client", "~> 0.23"
+  gem.add_dependency "google-api-client", ">= 0.30.4"
```

Comment:
Should this also have `< 1.0`?

Reply:
The REST library is never going to go 1.0 in its current form, but still, I think you're right, best to do it anyway just in case.